provider/rotation_data.go

```diff
@@ -0,0 +1,94 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/firehydrant/terraform-provider-firehydrant/firehydrant"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceRotation() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataFireHydrantRotation,
+		Schema: map[string]*schema.Schema{
+			// Required
+			"id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"team_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"schedule_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```

Comment:
Do we want to verify that the users are passing in valid GUIDs?

Reply:
The intent (from a tf user perspective) is that this is coming from a data resource that's getting the ID from the user email and it's either in their org or it isn't, regardless of format.  An extra test here to see if the ID matches the pattern we use just means that any change we make internally (UUIDv6 vs UUIDv7, for example) would need to be updated here as well, and that still doesn't guarantee that it'll match any of the users in their org.  So I'm inclined to make the provider be lighter weight if it isn't really adding value.

Reply:
Makes sense to me!